crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -3906,7 +3945,7 @@ fn create_window<T: UserEvent, F: Fn(RawWindow) + Send + 'static>(
   }
 
   #[cfg(desktop)]
-  if window_builder.center {
+  if window_builder.prevent_overflow.is_some() || window_builder.center {
     let monitor = if let Some(window_position) = &window_builder.inner.window.position {
       event_loop.available_monitors().find(|m| {
```

Comment:
I don't remember why we are doing it this way, but can't we change this to [`monitor_from_point`](https://docs.rs/tao/latest/tao/event_loop/struct.EventLoopWindowTarget.html#method.monitor_from_point)

Reply:
`window_builder.inner.window.position` is a `Position` here that we need to convert to physical position with scale factor so I don't think we can use `monitor_from_point` here, also I think `monitor_from_point` was introduced after this PR was made?

Reply:
maybe, let's keep it as is for now then